docs/data-sources/users.md

```diff
@@ -0,0 +1,39 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "twingate_users Data Source - terraform-provider-twingate"
+subcategory: ""
+description: |-
+  Users in Twingate can be given access to Twingate Resources and may either be added manually or automatically synchronized with a 3rd party identity provider. For more information, see see Twingate's documentation https://docs.twingate.com/docs/users.
+---
+
+# twingate_users (Data Source)
+
+Users in Twingate can be given access to Twingate Resources and may either be added manually or automatically synchronized with a 3rd party identity provider. For more information, see see Twingate's [documentation](https://docs.twingate.com/docs/users).
+
+## Example Usage
+
+```terraform
+data "twingate_users" "all" {}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Read-Only
+
+- `id` (String) The ID of this resource.
+- `users` (Block List) (see [below for nested schema](#nestedblock--users))
+
+<a id="nestedblock--users"></a>
+### Nested Schema for `users`
+
+Read-Only:
```

Comment:
`is_admin` should read `Indicates whether the User is an admin`
`role` should read `Indicates the User's role. Either ADMIN, DEVOPS, SUPPORT, or MEMBER.`

Reply:
updated